### ISSUE TITLE
Add unleash feature v2 tags

### ISSUE THREAD

**Issue by marylly** [philips-labs/terraform-provider-unleash#77]
🚀 Feature request

Description:
The ability to create, update and delete toggles tags

Is your proposal related to a problem?
Having the ability to filter toggles according to the same tags related to, it will allow for better toggles management

Describe the solution you'd like
```
resource "unleash_feature_v2" "with_env_strategies" {
  name               = "my_nice_feature"
  description        = "manages my nice feature"
  type               = "release"
  project_id         = "default"
  archive_on_destroy = false

  environment {
    name    = "production"
    enabled = false
  }

  environment {
    name    = "development"
    enabled = true

    strategy {
      name = "remoteAddress"
      parameters = {
        IPs = "189.434.777.123,host.test.com"
      }
    }
  }
  tags {
     type = "simple"
     value = "feature1"
  }
}
```

**Comment by jdelucaa** [comment]
Hey 👋 
Let me have a look and get back to you.
Thanks for suggesting this.

**Comment by jdelucaa** [comment]
Available on v0.3.7 🚀🚀🚀
Thanks for the help with the API client. Please let me know if that works for you.
